internal/server/data/migrations.go

```diff
@@ -935,8 +935,9 @@ ALTER TABLE ONLY user_public_keys
 
 CREATE UNIQUE INDEX IF NOT EXISTS idx_user_public_keys_user_fingerprint ON user_public_keys
     USING btree (fingerprint) WHERE (deleted_at IS NULL);
+
+ALTER TABLE identities ADD COLUMN IF NOT EXISTS ssh_username text
```

Comment:
Reminder to self, this is not safe if I merge #3595 first which added this migration. I'll need to create a new one.